### DISCUSSION TITLE
.env修改后无改变

### DISCUSSION THREAD

**Discussion by nljie1103** [imsyy/home#144] [General]
.env里面内容修改后，网页却没有改变，是咋回事啊，是不是有缓存，怎么清空缓存，我重启服务器没用

**Comment by imsyy** [comment]
改变的哪一项呀

**Reply by nljie1103** [reply]
网站的名字和高德的api，音乐id改了，改的都是引号里面内容

**Reply by imsyy** [reply]
> 网站的名字和高德的api，音乐id改了，改的都是引号里面内容

拉取一下最新版本试试，我这边开发环境测试正常

**Reply by nljie1103** [reply]
最新版？啥时候更新的，我昨天搭建的，怎么拉取最新版

**Reply by nljie1103** [reply]
包括我把跳转链接改了几个，也没用

**Comment by nljie1103** [comment]
应该不是缓存吧，我的重启好几遍了，nginx，yarn缓存都清了

**Reply by imsyy** [reply]
git pull 拉取更新，更改 env 文件后需要重新构建才能实现修改，只有开发环境下不用重新构建

**Reply by nljie1103** [reply]
root@localhost ~]# cd /www/wwwroot/home
[root@localhost home]# git pull git@github.com:imsyy/home.git
来自 github.com:imsyy/home
 * branch            HEAD       -> FETCH_HEAD
更新 acc8f54..177b539
error: Your local changes to the following files would be overwritten by merge:
        .env
        public/socialLinks.json
        src/components/Links/index.vue
Please, commit your changes or stash them before you can merge.
Aborting

**Reply by imsyy** [reply]
自行解决冲突后重新拉取

**Reply by nljie1103** [reply]
[root@localhost home]# git pull git@github.com:imsyy/home.git
来自 github.com:imsyy/home
 * branch            HEAD       -> FETCH_HEAD
Already up-to-date.

pull拉取后是最新的了，可页面还是你的页面，依旧不是我修改的内容

**Reply by nljie1103** [reply]
[root@localhost home]# yarn build
node: relocation error: /lib64/libnode.so.93: symbol FIPS_selftest, version OPENSSL_1_1_0g not defined in file libcrypto.so.1.1 with link time reference

我之前好像忘执行yarn build了。。。。。但是执行后提示这个

**Reply by RuSheng920** [reply]
你要先改再部署

**Comment by Sdongmaker** [comment]
![image](https://github.com/imsyy/home/assets/114125848/948fc465-412b-47c7-82f7-3a8002fdb24a)
hui会有这样的报错还有就是更改了不生效
![Uploading image.png…]()
